app/controllers/bill_details_controller.rb

```diff
@@ -0,0 +1,4 @@
+class BillDetailsController < ApplicationController
+  def new
```

Comment:
```suggestion
  def new; end
```
tương tự cho các chỗ khác